content/faq/publishing-standards.md

```diff
@@ -0,0 +1,21 @@
+---
+title: What are the recommended dimensions for thumbnails and covers?
+category: publisher
+order: 2
+---
+
+Even though LBRY does not enforce any file size, dimension, or aspect ratio properties, our applications are optimized to provide the best experience for viewing content thumbnails, channel covers, and channel profile thumbnails. If you are using the apps to upload the images, these uploads go through [spee.ch](https://spee.ch) which is limited to 10MBs per file. 
+
+### Supported file types
+A majority of image file types including jpg, png, and gif can be hyperlinked or uploaded. 
+
+### Content thumbnails
+When publishing content thumbnails, we recommend a size of 800x450 and/or an aspect ratio of 16:9.
```

Comment:
I do not want to recommend exact sizes. Recommend aspect ratio and minimum size only. 

If recommending a maximum size, recommend it in terms of file-size only.

Reply:
Applications can make their own choices on how to render thumbnails. Most applications choose to crop to the center when an image does not fit their desired ratios.

Reply:
Please adjust the help text in lbry-desktop as well.